models/yang/sonic/sonic-neigh.yang

```diff
@@ -1,61 +0,0 @@
-module sonic-neigh {
-    namespace "http://github.com/Azure/sonic-neigh";
-    prefix sneigh;
-
```

Comment:
Not sure why we deleted this file.

Reply:
It has been replaced by sonic-arp-ndp.yang

Reply:
it has been replaced by 'sonic-neighbors.yang'